stdlib/influxdata/influxdb/v1/v1.flux

```diff
@@ -11,6 +11,21 @@ fieldsAsCols = (tables=<-) =>
     tables
         |> pivot(rowKey:["_time"], columnKey: ["_field"], valueColumn: "_value")
 
+// FieldKeys returns the list of field keys in a given bucket.
+// The return value is always a single table with a single column, "_value".
+fieldKeys = (bucket, predicate=(r) => true, start=-30d) =>
+    from(bucket: bucket)
+      |> range(start: start)
+      |> filter(fn: predicate)
+      |> keep(columns: ["_field"])
+      |> group()
+      |> distinct(column: "_field")
```

Comment:
This is just a special case of `tagValues`, right? In other words, could you just define it like this:
```
fieldKeys = (bucket, predicate=(r) => true, start=-30d) =>
    tagValues(bucket: bucket, tag: "_field", predicate: predicate, start: start)
```

Reply:
Good call. Updated.